--- Granny and the Gators (Bally 1984) 1.1.vbs.original
+++ Granny and the Gators (Bally 1984) 1.1.vbs
@@ -1,27 +1,33 @@
 Option Explicit
-LoadVPM "01530000","granny.vbs",3.1
 
-
-Sub LoadVPM(VPMver,VBSfile,VBSver)
 	On Error Resume Next
-		If ScriptEngineMajorVersion<5 Then MsgBox"VB Script Engine 5.0 or higher required"
-		ExecuteGlobal GetTextFile(VBSfile)
-		If Err Then MsgBox"Unable to open "&VBSfile&". Ensure that it is in the same folder as this table."&vbNewLine&Err.Description:Err.Clear
-
-'**************************************************************************************************
-'********************************ACTIVATE BACKGLASS************************************************
-If ShowDT=true then
-		Set Controller=CreateObject("VPinMAME.Controller")
-	else
-		Set Controller=CreateObject("B2S.server") 
-End if
-'**************************************************************************************************
-
-		If Err Then MsgBox"Can't Load VPinMAME."&vbNewLine&Err.Description
-		If VPMver>"" Then If Controller.Version<VPMver Or Err Then MsgBox"VPinMAME ver "&VPMver&" required.":Err.Clear
-		If VPinMAMEDriverVer<VBSver Or Err Then MsgBox VBSFile&" ver "&VBSver&" or higher required."
+	ExecuteGlobal GetTextFile("controller.vbs")
+	If Err Then MsgBox "You need the controller.vbs in order to run this table, available in the vp10 package"
 	On Error Goto 0
-End Sub
+
+LoadVPM "01530000","Granny.vbs",3.1
+
+
+' Sub LoadVPM(VPMver,VBSfile,VBSver)
+' 	On Error Resume Next
+' 		If ScriptEngineMajorVersion<5 Then MsgBox"VB Script Engine 5.0 or higher required"
+' 		ExecuteGlobal GetTextFile(VBSfile)
+' 		If Err Then MsgBox"Unable to open "&VBSfile&". Ensure that it is in the same folder as this table."&vbNewLine&Err.Description:Err.Clear
+
+' '**************************************************************************************************
+' '********************************ACTIVATE BACKGLASS************************************************
+' If ShowDT=true then
+' 		Set Controller=CreateObject("VPinMAME.Controller")
+' 	else
+' 		Set Controller=CreateObject("B2S.server") 
+' End if
+' '**************************************************************************************************
+
+' 		If Err Then MsgBox"Can't Load VPinMAME."&vbNewLine&Err.Description
+' 		If VPMver>"" Then If Controller.Version<VPMver Or Err Then MsgBox"VPinMAME ver "&VPMver&" required.":Err.Clear
+' 		If VPinMAMEDriverVer<VBSver Or Err Then MsgBox VBSFile&" ver "&VBSver&" or higher required."
+' 	On Error Goto 0
+' End Sub
 
 
 
